pyvrp/search/tests/test_Route.py

```diff
@@ -285,46 +285,6 @@ def test_all_routes_overlap_with_maximum_tolerance_value(ok_small):
     assert_(route2.overlaps_with(route1, 1))
 
 
-def test_data_is_not_updated_until_update_call(ok_small):
```

Comment:
This was already a bit of a weird test, but such behaviour is now no longer allowed: accessing such data before calling uodate() triggers the asserts.